charts/area-chart/step4.md

```diff
@@ -1,61 +1,34 @@
-Now we'll add multiple datasets to the area chart...
+Now you will add tooltips to the area chart.
 
 ## Task
 
-1) Make sure the App.js file is still open
+1) Make sure the App.js file is still open.
 
 2) Locate the code that looks like the following:
 
 <pre class="file">
-&lt;ChartArea
-  data={[
-    { name: &#39;Cats&#39;, x: &#39;2015&#39;, y: 3 }, 
-    { name: &#39;Cats&#39;, x: &#39;2016&#39;, y: 4 }, 
-    { name: &#39;Cats&#39;, x: &#39;2017&#39;, y: 8 }, 
-    { name: &#39;Cats&#39;, x: &#39;2018&#39;, y: 6 }
-  ]} 
-/&gt;
+&lt;Chart
+  height={250}
+  width={600}
+&gt;
 </pre>
 
-3) Replace that section with the following code:
+3) Add the following property to that section:
 
 <pre class="file" data-target="clipboard">
-&lt;ChartGroup&gt;
-  &lt;ChartArea
-    data={[
-      { name: &#39;Cats&#39;, x: &#39;2015&#39;, y: 3 }, 
-      { name: &#39;Cats&#39;, x: &#39;2016&#39;, y: 4 }, 
-      { name: &#39;Cats&#39;, x: &#39;2017&#39;, y: 8 }, 
-      { name: &#39;Cats&#39;, x: &#39;2018&#39;, y: 6 }
-    ]}
-    interpolation="basis"
+containerComponent={
+  &lt;ChartVoronoiContainer
+    constrainToVisibleArea
+    labels={({ datum }) =&gt; `${datum.name}: ${datum.y}`}
   /&gt;
-  &lt;ChartArea 
-    data={[
-      { name: &#39;Dogs&#39;, x: &#39;2015&#39;, y: 2 }, 
-      { name: &#39;Dogs&#39;, x: &#39;2016&#39;, y: 3 }, 
-      { name: &#39;Dogs&#39;, x: &#39;2017&#39;, y: 4 }, 
-      { name: &#39;Dogs&#39;, x: &#39;2018&#39;, y: 5 },
-      { name: &#39;Dogs&#39;, x: &#39;2019&#39;, y: 6 }
-    ]}
-    interpolation="basis"
-  /&gt;
-  &lt;ChartArea 
-    data={[
-      { name: &#39;Birds&#39;, x: &#39;2015&#39;, y: 1 }, 
-      { name: &#39;Birds&#39;, x: &#39;2016&#39;, y: 2 }, 
-      { name: &#39;Birds&#39;, x: &#39;2017&#39;, y: 3 }, 
-      { name: &#39;Birds&#39;, x: &#39;2018&#39;, y: 2 },
-      { name: &#39;Birds&#39;, x: &#39;2019&#39;, y: 4 }
-    ]}
-    interpolation="basis"
-  /&gt;
-&lt;/ChartGroup&gt;
+}
 </pre>
 
-- The `ChartGroup` is the React Component used to apply properties to multiple `ChartArea` components
+<strong>Note:</strong> The `constrainToVisibleArea` property will alter the position of the tooltip so that it exactly fits within the rendered Scalable Vector Graphic (SVG).

```

Comment:
I thought we had spoken about breaking up these properties into individually steps? and then defining the properties with each step.

Reply:
noted....